models/stg_facebook_pages.yml

```diff
@@ -212,7 +190,7 @@ models:
         description: The total number of views on the post longer than 10 seconds when the post was clicked to play
       - name: video_views_10s_organic
         description: The total number of organic views on the post longer than 10 seconds
-      - name: video_views_10s_paid
+      - name: video_views_10_s_paid
```

Comment:
This is how it's spelled in the model, so thought it was better to update here than in the model.